semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +123,38 @@ p strong {
   margin-left: 10px;
 }
 
+/* Application home */
+#application-area {
+  font-family: @userAreaFont;
+  display: flex;
+  flex-direction: column;
+}
+
+.company-title .h4 {
+  height: 45px;
+  width: 185px;
+  letter-spacing: @letterSpacingRegular;
+  line-height: @lineHeightBig;
+}
+
+/* TODO: Add issue to trim if app name is too big*/
+.back {
+  align-self: flex-start;
+  width: 132px;
```

Comment:
Just set a max-width so it can accommodate longer text?

Reply:
Ok. Changing that along with the change from **Button** to **Label** : )

Reply:
Done: a58df28